jobs/config.json

```diff
@@ -5275,6 +5275,24 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7": {
+    "args": [
+      "--cluster=",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7.env",
+      "--kubeadm=ci",
+      "--kubernetes-anywhere-kubernetes-version=latest-1.6",
+      "--deployment=kubernetes-anywhere",
+      "--mode=local",
+      "--timeout=300m",
+      "--extract=ci/latest-1.6",
```

Comment:
Would be nice to have proper docs of that, even if they were kubeadm-e2e-specific ;)